bin/kano-jupyter

```diff
@@ -87,6 +87,7 @@ if __name__ == '__main__':
     args = docopt.docopt(__doc__)
 
     if args['run']:
+        os.system('kdesk-hourglass-app "Chromium"')
```

Comment:
Chromium?

Reply:
It means start the hourglass until a window comes up with the word "Chromium" somewhere on it's titlebar.